### DISCUSSION TITLE
irregular cache file name

### DISCUSSION THREAD

**Discussion by sepehrhakimi90** [nginx/nginx#329] [Q&A]
I have some files in the cache_path with md5 of the cache key and some random numbers. this is ab example: 16ca7e9da4fc97802235c7c2fddefef2.0065803486

it would be great if you help me to know in which cases this names extend?

Also i found that the cache status stuck in updating since then.

**Comment by Maryna-f5** [comment]
This discussion appears to focus on the configuration, troubleshooting or general questions around running NGINX, which is not the intended use for GitHub discussions. 

We encourage you to visit [https://community.nginx.org ](https://community.nginx.org/)and share your discussion there as the NGINX Community Forum is the most appropriate place to engage for these types of questions.
